tests/functional/firefox/test_mobile.py

```diff
@@ -8,61 +8,32 @@
 from pages.firefox.mobile import FirefoxMobilePage
 
 
+# mobile - send to device
 @pytest.mark.nondestructive
 def test_get_firefox_send_to_device_success(base_url, selenium):
     page = FirefoxMobilePage(selenium, base_url).open()
-    modal = page.click_get_firefox_header_button()
+    modal = page.click_get_firefox_button()
     assert modal.is_displayed
     assert not page.is_firefox_qr_code_displayed
     send_to_device = page.send_to_device
     send_to_device.type_email('success@example.com')
     send_to_device.click_send()
     assert send_to_device.send_successful
-    modal.close()
 
 
 @pytest.mark.nondestructive
 def test_get_firefox_send_to_device_fails_when_missing_required_fields(base_url, selenium):
     page = FirefoxMobilePage(selenium, base_url).open()
-    modal = page.click_get_firefox_header_button()
+    modal = page.click_get_firefox_button()
     assert modal.is_displayed
     with pytest.raises(TimeoutException):
         page.send_to_device.click_send()
 
-
+# mobile - qr code
 @pytest.mark.nondestructive
-def test_get_firefox_qr_code_locale(base_url, selenium):
+def test_get_firefox_qr_code(base_url, selenium):
     page = FirefoxMobilePage(selenium, base_url, locale='it').open()
```

Comment:
Ok so this is interesting - even though this passes against run-integration-tests (where Dev=True and all locales are active), once this gets to stage/prod this test will likely fail, since the Italian translation for this page [isn't yet active](https://github.com/mozilla-l10n/www.mozilla.org/blob/master/it/firefox/mobile-2019.lang).

For now, I think it's probably best we skip this test with a marker `@pytest.mark.skip(reason='Page is not yet active for Italian locale.')`. We can then remove the marker later once the locale is active.

Reply:
I just switched to an already active local that meets the requirements